src/chocolatey/infrastructure.app/services/ChocolateyPackageInformationService.cs

```diff
@@ -29,6 +29,7 @@ public class ChocolateyPackageInformationService : IChocolateyPackageInformation
         private readonly IRegistryService _registryService;
         private readonly IFilesService _filesService;
         private const string REGISTRY_SNAPSHOT_FILE = ".registry";
+        private const string REGISTRY_SNAPSHOT_BACKUP_FILE = ".registry.backup";

```

Comment:
Is backup a good name here? I want this to signify that the file is bad in some way. Perhaps `.registry.bad`?